libs/core/langchain_core/utils/_merge.py

```diff
@@ -30,6 +30,8 @@ def merge_dicts(left: Dict[str, Any], right: Dict[str, Any]) -> Dict[str, Any]:
             )
         elif isinstance(merged[k], str):
             merged[k] += v
+        elif isinstance(merged[k], int):
+            merged[k] += v
```

Comment:
wait sorry we don't want to add up ints either. its ambiguous how to merge generic ints which is why they're explicitly not supported

Reply:
> wait sorry we don't want to add up ints either. its ambiguous how to merge generic ints which is why they're explicitly not supported

Okay, let me close this PR first. Thanks @baskaryan